src/components/ConfirmationMessage.tsx

```diff
@@ -20,8 +20,8 @@ const CenteredParagraph = styled(Paragraph)`
 
 interface ConfirmationMessageProps {
   title: string;
-  message: string;
-  details: string;
+  message: string | JSX.Element;
+  details: string | JSX.Element;
```

Comment:
Hmm I'm not sure if I'm happy with this change but it depends how you use it later.

Reply:
Some of the confirmation message pages may have a "click here to go back to the homepage" or something, so the type flexibility makes it easier to do that

Reply:
Seems reasonable to me